Add New Contributors Closing Issues metric (#277)
chaoss/wg-evolution: focus_areas/community_growth.md

[New Contributors](../metrics/contributors-new.md) | What is the overall number of new contributors?
[New Contributors of Commits](../metrics/pull-requests-merge-contributor-new.md) | What is the number of persons contributing with an accepted commit for the first time?
[New Contributors of Issues](../metrics/issues-first-time-opened.md) | What is the number of persons opening an issue for the first time?

> [New Contributors Closing Issues](../metrics/New_Contributors_Closing_Issues.md) | What is the number of persons closing an issue for the first time?

Reviewer: update the focus aread to have the same question as the metric

```suggestion
[New Contributors Closing Issues](../metrics/New_Contributors_Closing_Issues.md) | How many contributors are closing issues for the first time?
```
